lib/trino-orc/src/main/java/io/trino/orc/stream/LongInputStreamV1.java

```diff
@@ -58,13 +58,13 @@ private void readValues()
             numLiterals = control + MIN_REPEAT_SIZE;
             used = 0;
             repeat = true;
-            delta = input.read();
+            int delta = input.read();
             if (delta == -1) {
                 throw new OrcCorruptionException(input.getOrcDataSourceId(), "End of stream in RLE Integer");
             }
 
             // convert from 0 to 255 to -128 to 127 by converting to a signed byte
-            delta = (byte) delta;
+            this.delta = (byte) delta;
```

Comment:
the code here was legit (not a "SelfAssignment"). the `delta` local variable was introduced as a workaround only

have you reported this to errorprone already?